src/profile-logic/transforms.js

```diff
@@ -1333,6 +1337,18 @@ export function restoreAllFunctionsInCallNodePath(
   return newCallNodePath.reverse();
 }
 
+export function funcToImplementation(
+  thread: Thread,
+  funcIndex: IndexIntoFuncTable
+): Implementation {
+  if (FUNC_MATCHES.cpp(thread, funcIndex)) {
+    return 'cpp';
+  } else if (FUNC_MATCHES.js(thread, funcIndex)) {
+    return 'js';
+  }
+  return 'unknown';
```

Comment:
I didn't know what this case implies, hence 'unknown'. But perhaps we know what this is?

Reply:
These are probably JIT code, that is generated by the JS engine, but could also be unsymbolicated native code, e.g. from a third party library like Flash.